temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
```

Comment:
**CODE_QUALITY**: The import statement for `pdb` seems unnecessary for the current implementation. Consider removing it if not required.
```
import pdb
```